src/test/java/seedu/address/model/person/StatusTest.java

```diff
@@ -0,0 +1,54 @@
+package seedu.address.model.person;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+import seedu.address.testutil.Assert;
+
+public class StatusTest {
+
+    @Test
+    public void valid_status_values() {
```

Comment:
Consider renaming your tests to a more significant name. i.e. methodName_purposeOfTest()